router/dealer.go

```diff
@@ -646,6 +646,18 @@ func (d *dealer) syncCall(caller *wamp.Session, msg *wamp.Call) {
 	storedInvocationID, ok := d.invocationByCall[callReqID]
 	isInProgress, _ := msg.Options[wamp.OptProgress].(bool)
 
+	if isInProgress && !caller.HasFeature(wamp.RoleCaller, wamp.FeatureProgCallInvocations) {
+		// The Caller did not announce the progressive call invocations feature during the HELLO handshake.
+		abortMsg := wamp.Abort{Reason: wamp.ErrProtocolViolation}
+		abortMsg.Details = wamp.Dict{}
+		abortMsg.Details[wamp.OptMessage] = "Peer is trying to use Progressive Call Invocations while it was not " +
+			"announced during HELLO handshake"
+		caller.Peer.Send(&abortMsg)
+		//caller.Peer.Close()
+
+		return
+	}
```

Comment:
@gammazero hi! I need your help a bit! 

During fix/testing I faced a problem: there are some cases when I need to abort peer session from the `sync*` methods. Like here: client is making an RPC call (`func (d *dealer) call(caller *wamp.Session, msg *wamp.Call) `)  which in turn pushes a func into the dealer's action channel:
```
	d.actionChan <- func() {
		d.syncCall(caller, msg)
	}
```

And within `syncCall` in some cases, we fall into protocol violation and the session needs to be aborted. It looks like I'm doing it wrong (see the code above).... so I'm receiving: `panic: send on closed channel`. 

Can't figure out how to make that properly.... Maybe you can suggest how?
There is no problem with doing it from the client side outside the dealer, but from here - I'm stuck...